src/graphql/schema.gql

```diff
@@ -433,6 +434,7 @@ type Proposal {
   start: Int!
   end: Int!
   quorum: Float!
+  quorumType: String!
```

Comment:
Why is this one using `String!`, and the one from SpaceVoting using just `String` ?

Reply:
Updated, Thank you!